bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly.properties

```diff
@@ -246,9 +247,9 @@ channel-type.shelly.temperature3.description = Temperature of external Sensor #3
 channel-type.shelly.temperature4.label = Temperature 4
 channel-type.shelly.temperature4.description = Temperature of external Sensor #4
 channel-type.shelly.temperature5.label = Temperature 5
-channel-type.shelly.temperature6.description = Temperature of external Sensor #5
+channel-type.shelly.temperature5.description = Temperature of external Sensor #5
 channel-type.shelly.targetTemp.label = Target Temperature
-channel-type.shelly.targetTemp.description = Target Temperature in �C to be reached in auto-temperature mode
+channel-type.shelly.targetTemp.description = Target Temperature in \uFFFDC to be reached in auto-temperature mode
```

Comment:
Is this change required ?

Reply:
I just edited the message to include a °C (as requested). I think \uFFFDC was added by IDE or man:spotless:apply?